indico/modules/events/contributions/templates/forms/contribution_person_widget.html

```diff
@@ -82,8 +80,8 @@
 
             function renderPerson(person) {
                 var $personRow = $('<div>').addClass('person-row');
-                var $personName = $('<span>').text(person.name);
-                var $personStatus = $('<span>').addClass('person-status');
+                var $personName = $('<div>').addClass('name').text(person.name);
+                var $personStatus = $('<div>').addClass('person-status');
                 var $personRoles = $('<span>').addClass('person-roles');
                 var $personButtons = $('<span>').addClass('person-buttons');
                 var $buttonRemove = $('<a>').addClass('i-button-icon danger icon-close').attr('title', $T.gettext("Remove person"));
```

Comment:
At least for multi-line statements it's much more readable with `{..}` than having wrapped chain calls. For one-line statements... let's let everyone use what he prefers? Anyway, it's not really related to this PR - the comment is on a line that's just context ;)